__tests__/app/models/session.jest.js

```diff
@@ -0,0 +1,30 @@
+const db = require('../../../app/models/index')
+const moment = require('moment')
+
+describe("Session model", () => {
+    let Session
+
+    beforeAll(async() => {
+        await db.sequelize.sync() // connect to the database
+        session = await db.Session.create()
+    })
+
+    describe("Session.create", () => {
+        it ("Should create a valid session record with default time in expires (Now + 4 hours)", async () => {
+            expect(moment(session.expires).Empty).toBeFalsy()
+        })
+
+        it ("Should return false, as we just made the session", async() => {
+            expect(session.checkIfExpired()).toBeFalsy()
+        })
+    })
+    
+
+    afterAll(async () => {
```

Comment:
Same with this - moved it to an after-the-fact closing of connection. Should probably just call "session.destroy()" so the session you created in the beforeAll is deleted